consensus/babe/types.go

```diff
@@ -148,13 +169,19 @@ func (n *NextEpochDescriptor) Encode() []byte {
 }
 
 // Decode sets the NextEpochDescriptor to the SCALE decoded input.
+// TODO: change to io.Reader
```

Comment:
Can this be addressed here or should we open a separate issue?

Reply:
#611